client/my-sites/plans-features-main/hooks/use-non-owner-handler.ts

```diff
@@ -0,0 +1,139 @@
+import { type Member, type Plans } from '@automattic/data-stores';
+import usePurchasesQueryKeysFactory from '@automattic/data-stores/src/purchases/queries/lib/use-query-keys-factory';
+import { getUseSitePurchasesOptions } from '@automattic/data-stores/src/purchases/queries/use-site-purchases';
+import useSiteQueryKeysFactory from '@automattic/data-stores/src/site/queries/lib/use-query-keys-factory';
+import { getUseSiteUserQueryOptions } from '@automattic/data-stores/src/site/queries/use-site-user-query';
+import { useStillNeedHelpURL } from '@automattic/help-center/src/hooks';
+import { HELP_CENTER_STORE } from '@automattic/help-center/src/stores';
+import { useQueryClient } from '@tanstack/react-query';
+import { useDispatch } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+import { useTranslate, type LocalizeProps } from 'i18n-calypso';
+
+export function useNonOwnerHandler( {
+	siteId,
+	currentPlan,
+}: {
+	siteId?: number | null;
+	currentPlan: Plans.SitePlan | undefined;
+} ) {
+	const { setShowHelpCenter, setInitialRoute, setOdieBotNameSlug, setOdieInitialPromptText } =
+		useDispatch( HELP_CENTER_STORE );
+	const translate = useTranslate();
+
+	const { url: stillNeedHelpUrl } = useStillNeedHelpURL();
```

Comment:
You shouldn't need to use this. This was something leftover when we had multiple support options that fluctuated based on the users support level. You just need to set the path to: `/odie` to get it to start on Wapuu. Not using a direct path to Wapuu could leave to inconsistencies.

I would also say that adding the bot name as a query arg to the path would be a much more simple solution. `/odie?bot=wpcom-plan-support`.

Reply:
Thanks @arcangelini, I have replaced `useStillNeedHelpURL` with the string `/odie`.

Passing the bot name as query arg sounds like a good idea to me. Related to your other comment (https://github.com/Automattic/wp-calypso/pull/92015#discussion_r1689639693), we could pass both `bot` and `initialPromptText` as query args and bypass the help-center data store altogether.

The current approach was largely based on this recommendation: https://github.com/Automattic/wp-calypso/pull/91925#issuecomment-2202901733

Thanks again for the feedback! And please let me know what you think about the above.